recipes/s2n/all/test_package/test_package.c

```diff
@@ -0,0 +1,9 @@
+#include <aws/compression/compression.h>
+
+int main() {
+    struct aws_allocator *allocator = aws_default_allocator();
```

Comment:
I think this is the test from `aws-c-compression`.

Reply:
Yes I changed computer so I push this draft build from `aws-c-compression` recipe.